Fits/T1 EWAR Cruisers/[NVY-30] Blackbird.md

```diff
@@ -0,0 +1,51 @@
+# Blackbird
+
+Entry EWAR Cruiser
+
+## Description
+
+Utilizes Electronic Counter Measures (ECM) to disrupt hostile DPS or Logistics. This ship is entirely reliant on range to avoid incoming DPS so do not burn close unless absolutely necessary.
+
+If you know what the enemy is going to be fielding on grid, use the list below to pick the appropriate ECM modules since they are stronger against their specific targets.
+
+Ladar - Minmatar, Angel Cartel.
+Radar - Amarr, Blood Raiders, Sansha's Nation, Triglavian Collective.
+Magnometric - Gallente, ORE, Sisters of Eve.
+Gravimetric - Caldari, Guristas, Mordu's Legion.
+
+## Fit
+
+```
+[Blackbird, [NVY-30] Blackbird]
```

Comment:
Not really worth it as an armor ship, immediately gets deleted, doesn't have the tank like a Scorpion. Griffin is better if you fit Blackbird like this.  Can be decent as a shield brick, though.

```
[Blackbird, [ADV-30] Blackbird]

Damage Control II
Mark I Compact Power Diagnostic System
Mark I Compact Power Diagnostic System

50MN Quad LiF Restrained Microwarpdrive
Enduring Multispectrum Shield Hardener
Enduring Multispectrum Shield Hardener
Large F-S9 Regolith Compact Shield Extender
Large F-S9 Regolith Compact Shield Extender
Deluge Enduring Burst Jammer

[Empty High slot]
[Empty High slot]
[Empty High slot]
[Empty High slot]

Medium Core Defense Field Extender I
Medium Core Defense Field Extender I
Medium Core Defense Field Extender I


FZ-3a Enduring Gravimetric ECM x1
Gloom Enduring Radar ECM x1
Languid Enduring Ladar ECM x1
Morpheus Enduring Magnetometric ECM x1
```